### PR TITLE
Added blog post for week 14

### PR DESCRIPTION
**Pull request by m-agour**
My blog post for week 14

### CODE REVIEW

**Review by ganimtron-10 — COMMENTED** [review]
Hello @m-agour ,
Nice Blog. LGTM.

File: docs/source/posts/2022/2022-09-28-week-14-mohamed.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+What is coming up next week?
+----------------------------
+
+- Add unit testing for the ``Animation``, document it well, and implement more properties suggested by Shivam (@xtanion).
+
+- Modify `Add Timelines to ShowManager directly`_ PR to allow adding ``Animation`` to the ``ShowManager`` as well.
+
+- Update tutorials to adapt to all the new changes in the ``animation`` module.
```
Comment (xtanion) [inline docs/source/posts/2022/2022-09-28-week-14-mohamed.rst:27]: ``Animation`` module, to keep things consistent

**Review by xtanion — COMMENTED** [review]
Nice blog @m-agour, Everything looks great. PTAL at my comment below

File: docs/source/posts/2022/2022-09-28-week-14-mohamed.rst (hunk 1 above)
Comment (m-agour) [reply docs/source/posts/2022/2022-09-28-week-14-mohamed.rst:27]: I meant here the whole `fury.animation` module, not only Animation class. According to pep, module name should be in lowercase

**Review by skoudoro — COMMENTED** [review]
All good, merging

**Pull request merged**